### ISSUE TITLE
kube-down prints strange etcd-related errors 

### ISSUE THREAD

**Issue by mwielgus** [kubernetes/kubernetes#35787]
```
ERROR: (gcloud.compute.ssh) could not parse resource: []
Removing etcd replica, name: , port: 2379, result: 1
ERROR: (gcloud.compute.ssh) could not parse resource: []
Removing etcd replica, name: , port: 4002, result: 1
```

cc: @jszczepkowski @fgrzadkowski

**Comment by jszczepkowski** [comment]
Works for me.

I get:
```
Warning: Permanently added 'compute.2343716106104233014' (RSA) to the list of known hosts.
Removing etcd replica, name: kubernetes-master, port: 2379, result: 0
Removing etcd replica, name: kubernetes-master, port: 4002, result: 0
```

In your case, probably the master was already removed, so, ssh failed.

**Comment by jszczepkowski** [comment]
@mwielgus 
Can you please give steps to reproduce it, or close it if reproduction is not possible.

**Comment by bgrant0607** [comment]
See this in PRs:
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/35797/pull-kubernetes-e2e-gce-etcd3/6236/?log#log

**Comment by bgrant0607** [comment]
Looks like this was fixed by #35081
